src/Chirp.Infrastructure/AuthorRepository.cs

```diff
@@ -174,6 +174,14 @@ public async void DeleteUser(string name, string email)
         Console.WriteLine(r);
     }
 
+    public async void DeleteFollowing(string name)
+    {
+        AuthorDTO a = await GetAuthorByName(name);
+        var r = _cheepDbContext.Followings.Where(follow => follow.AuthorId == a.AuthorId).ExecuteDeleteAsync().Result;
+        var r2 = _cheepDbContext.Followings.Where(follow => follow.FollowId == a.AuthorId).ExecuteDeleteAsync().Result;

```

Comment:
Shouldn't these use await? What do we use the result for (r,r2)? are they even needed?